App.js

```diff
@@ -19,14 +18,7 @@ import AddSiteScreen from './src/screens/AddSiteScreen';
 import MoreScreen from './src/screens/MoreScreen';
 import AuthScreen from "./src/screens/AuthScreen";
 
-const store = createStore(
-    reducers,
-    {},
-    compose(
-        applyMiddleware(ReduxThunk)
-    ));
-
-
+import {store, persistor} from './src/store';
```

Comment:
Check the comments in the `store.js` file, we now export two things: the `'store` itself, and then the `persistor` which helps us persist the store's state.